src/django/pfb_analysis/management/commands/import_crash_data.py

```diff
@@ -0,0 +1,72 @@
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+import boto3
+import csv
+import logging
+import shutil
+import tempfile
+
+from pfb_analysis.models import Crash
+
+
+logger = logging.getLogger(__name__)
+
+def download_csv(csv_key):
+    tmpdir = tempfile.mkdtemp()
+    s3_client = boto3.client('s3')
+    s3_client.download_file(settings.AWS_STORAGE_BUCKET_NAME,
+                            csv_key,
+                            tmpdir+'/'+'crashes.csv')
+    return tmpdir
```

Comment:
A few notes on the downloader:
- Since the file compresses so well (3.2MB for the .zip vs 58MB for the .csv) it would be nice to store and download it in zipped form.  And extracting a zip file is pretty easy in Python.
- Since the filename is hard-coded, I'm not sure there's much value in passing some of these parameters around.  I would be inclined to just hard-code the names where they're needed and only pass values for things that aren't know ahead of time (like the tmpdir) or are used repeatedly.

Here's a version of `download_csv` that makes both those changes:
```python
def download_csv():
    tmpdir = tempfile.mkdtemp()
    zipfile_path = os.path.join(tmpdir, "crashes.zip")
    s3_client = boto3.client('s3')
    s3_client.download_file(settings.AWS_STORAGE_BUCKET_NAME,
                            "data/crashes.zip",
                            zipfile_path)
    with zipfile.ZipFile(zipfile_path, "r") as zip_ref:
        zip_ref.extractall(tmpdir)
    return os.path.join(tmpdir, "crashes.csv")
````

So then in `import_csv` you could just do `csv_path = download_csv()` and hand the resulting path to the `open()` call.  I think that reduces the overall amount of hard-coding and makes it a bit more contained without actually reducing flexibility.